Add content for new notebook page (#27382)
DataDog/documentation: content/en/notebooks/notebooks_new.md

## Editing a notebook

With Notebooks, you can start typing using markdown shortucts such as `#` for headers or ``` for code blocks. Notebooks comprise cells that can contain text or graphs. A new cell is created every time you hit `enter`. Notebooks automatically save your text content as you type. For graph cells, save all edits in the graph editor for the widget to be saved in the notebook.

### Cell types

Notebooks support the following cell types:
- [Graphs](#graphs-in-notebooks)
- Images
- Headers (H1 - H3)
- Lists (bullet lists, numbered lists, and checklists)
- Codeblocks
- Quotes
- [Markdown](#markdown-cells)

### Graphs in notebooks

Notebooks support the following graph types:

- [Timeseries][3] 
- [Top List][4] 
- [Table][5]
- [Query Value][6]
- [Heatmap][7]
- [Distribution][8]
- [List][9]
- [Profiling Flame Graph][10]
- [Funnel][11]
- [Pie][12]
- [Treemap][13]
- [Geomap][14]
- [SLO][15]
- Scatterplot  
- Change

The following graph types are expected to be available in Notebooks soon:
- Alert graph
- Alert value
- Check Status
- Retention
- Sankey
- Monitor Summary
- SLO List
- SLO Summary
- Host Map
- Topology Map

> - Service Summary

Author: Thanks Janine! Yes, they requested this info. For capitalization, I try to capitalize the "product" reference, but lower-case the customers' usage if that makes sense. I agree with your changes on this for the most part!

In a Notebook graph cell, hover over the widget to display options for editing and configuring the graph. 

> To edit the query or configure the graph's display, click the pencil icon or Command+click to open the full graph editor. You can adjust the local timeframe or link the graph to the global notebook time by clicking the clock icon.

Author: I wasn't sure either, and I'd never used the <kbd> before, today I learned!

### Table of Contents 

Notebooks automatically generate a table of contents from any headers you insert into your document. You can create a header using the markdown shortcut `#` or by selecting text and selecting Header in the toolbar.

### Notebook tags

{{< img src="/notebooks/notebooks_new/notebook_tags.png" alt="Notebook tag options to favorite a notebook, add a team, or add a type" style="width:80%;" >}}

| Tag action                | Description                                                                                                          |
|------------------------|----------------------------------------------------------------------------------------------------------------------|
| **Favorite a Notebook**| Favorite a notebook to pin it to the top of your results in the Notebook List page. To toggle a notebook as a favorite, click the star icon in the header of the Notebook. |
| **Tag by Team**        | Tagging a Notebook with a team allows you to use it as a filter when searching for a Notebook. You can tag a Notebook with up to 5 teams. To tag, click on the "team" option in the Notebook header and select the desired teams.  |

> | **Tag by Type**        | You can tag your notebooks with type tags for easier searching, such as: Postmortem, Runbook, Investigation, Documentation, Report. To tag, select Type at the top of the Notebook. |

Author: No, the type UI is different, there is a set list that you select from: 
![image](https://github.com/user-attachments/assets/7b463bfe-fe9d-4465-913d-060e5a8dcb8f)

|--------------------------|----------------------------------------------------------------------------------------------------------------------|
| **Navigating to Comments** | Saved comments appear in the right margin of the notebook. Click a comment highlight in the text to align it, or click a comment in the margin to scroll to its location. |
| **Responding to Comments** | Respond to comments by clicking on them in the right margin, which opens a comment box. You can write text, `@mention` a Datadog user, or resolve a comment by clicking **Resolve**. |
| **Linking to Comments**    | Link to a specific comment by clicking the link icon in the top right corner of the comment to copy its link.      |
| **Editing or Deleting Comments** | Edit or delete your comments by clicking the three-dot ellipsis menu in the top right corner of the comment.                 |
| **Comment Notifications** | By default, email notifications are sent to the notebook author for new comments by others. Users in a comment thread receive notifications for each reply. To adjust notifications, in the cog menu, select **Notifications**. |

## Multiplayer experience in Notebooks

> Notebooks are collaborative, allowing multiple users to edit simultaneously. If a collaborator opens your document, their cursor will appear inside the document. Hover over a cursor to see the collaborator's name.

Reviewer: Are "notebooks" and "documents" different things in notebooks? I'm wondering if we should get rid of "document" throughout if it doesn't actually mean something else.
```suggestion
Notebooks are collaborative, allowing multiple users to edit simultaneously. If a collaborator opens your document, their cursor appears inside the document. Hover over a cursor to see the collaborator's name.
```

<!-- TODO Add updated image from new notebooks -->

### Widgets

> When another user is editing a widget, an outline appears to indicate this. Since widgets are saved as last write wins, avoid editing a widget that someone else is working on.

Reviewer: Small rephrase so this doesn't read as "the outline appears to be indicating this, but it might actually be indicating that"
```suggestion
When another user is editing a widget, an outline appears around the widget. Since widgets are saved as last write wins, avoid editing a widget that someone else is working on.
```

<!-- TODO Add updated image from new notebooks -->

#### Presence

At the top of the document you can see a a row avatar images of all the users currently viewing the document. Hover over an avatar to see the name of the associated collaborator.

<!-- TODO Add updated image from new notebooks -->

## Configuring a notebook

### Template variables

Notebooks support template variables. Dynamically scope visualizations by adding and selecting template variable values. For more information, see [Template Variables][17].

### Time controls

By default, all graph cells are linked to the global time frame set in the notebook header.

> To view a different time frame, select an option in the global time picker, or scrub on a graph directly. The notebook URL will update to reflect this new time frame without saving it to the notebook.

Reviewer: ```suggestion
To view a different time frame, select an option in the global time picker, or scrub on a graph directly. The notebook URL updates to reflect this new time frame without saving it to the notebook.
```

**Note**: Clicking and dragging to zoom in on a graph does not unlock the cell from the global time. It changes the notebook's global time instead.

<!-- TODO Add updated image from new notebooks -->

To save this time as the notebook's default, click **Set Default Time**. To reset your global time back to the previously saved default global time, click the reset button.

Individual cells can be unlinked from the global time and set to an independent time frame.

<!-- TODO Add updated image from new notebooks -->

To view a different time frame on a single cell, edit the cell and use the toggle to unlink it from Global Time. Change the time frame using the time picker or by scrubbing on the graph. Changes made in edit mode are saved automatically when you click **Done**. To discard your changes, click **Cancel** instead of **Done**.

### Modes

You can switch between modes from within the notebook by selecting the dropdown at the top right of your notebook.

- **Editing**: Make changes to the notebook.
- **Viewing**: Contents are Read Only, preventing users from making unwanted edits to existing configurations and information.

### Version history

> From a notebook, click the **Configure** icon and click **Version history** to open the Version History side panel. You can preview, restore, or clone your notebook's version history. For more information, see the [Version History guide][18].

Reviewer: Is the **Configure** icon different from the cog icon? If they're the same thing, I like how you referred to it here - I like referring to what icons are for, so people can think about the objectives they're trying to accomplish, instead of looking for specific icons that can be harder to remember if there are many of them.

 You can preview the existing snapshot on any fixed-time graph by hovering over the camera icon while in edit mode.

To share a version of your notebook with snapshots, from the cog menu, click **View snapshots**. Copy the URL, or append `&view=snapshots` to the URL of any notebook that has snapshots enabled.

### Permissions

By default, all users have full access to notebooks.

Use granular access controls to limit the [roles][19] that may edit a particular notebook:
1. While viewing a notebook, click on the cog in the upper right. The settings menu opens.
1. Select **Permissions**.
1. Click **Restrict Access**.
1. The dialog box updates to show that members of your organization have **Viewer** access by default.
1. Use the dropdown to select one or more roles, teams, or users that may edit the notebook.
1. Click **Add**.
1. The dialog box updates to show that the role you selected has the **Editor** permission.

> 1. Click **Save**

Reviewer: ```suggestion
1. Click **Save**.
```

**Note:** To maintain your edit access to the notebook, the system requires you to include at least one role that you are a member of before saving. 

> To restore general access to a notebook with restricted access, follow the steps below:

Reviewer: As above, do you need Editor access for this?

1. While viewing the notebook, click on the cog in the upper right. The settings menu opens.
1. Select **Permissions**.
1. Click **Restore Full Access**.
1. Click **Save**.

## Finding notebooks

The [Notebooks List][1] page is the place to find all of your Notebooks.

<!-- TODO Add updated image from new notebooks -->

### Search

> The search field supports document text search. Type what you're looking for and the notebooks below will list the results.

Reviewer: As above, is a document a different thing here?

Author: ```suggestion
The search field supports text search. Type your query to display the relevant notebooks as results.
```

### Filtering

You can filter notebooks with the following methods:
| Filter Type      | Description                                                                 |
|------------------|-----------------------------------------------------------------------------|
| **Author**       | To filter by author, select the author dropdown and enter names to filter by. |
| **Team**         | To filter by team, select the team dropdown and enter team names to filter by. |
| **Notebook Type**| Filter by investigation, postmortem, runbook, report, or documentation.     |
| **Modified Date**| Filter based on how recently a notebook was edited using the modified date dropdown. |

There are also quick filters to access your notebooks and notebooks tagged with your teams.

### Jump back in

If no filters are enabled, the Jump Back In section appears, displaying the most recent notebooks you've viewed or edited.

<!-- TODO Add updated image from new notebooks -->

### Sorting notebooks

You can sort notebooks by selecting the ⭐, details, or modified headers to sort by those values.

## API access

> Notebooks APIs are currently being updated to support the new Notebooks product. Creating notebooks through the API places them in "compatibility mode," supporting only markdown and widget cells. 

Author: Leaving capitalized as a reference to the product

> You cannot read or edit new notebooks using the notebooks API; attempts to do so will result in a 400 error.

Reviewer: ```suggestion
You cannot read or edit new notebooks using the notebooks API; attempts to do so result in a 400 error.
```
